src/index.css

```
@@ -2,14 +2,53 @@
+
   html {
-    @apply bg-black text-white antialiased;
+    @apply bg-black text-white antialiased scroll-smooth;
   }
+
   body {
-    @apply min-h-screen m-0;
+    @apply min-h-screen m-0 bg-black text-white;
+    background-image:
+      radial-gradient(circle at top, rgba(255, 99, 71, 0.18), transparent 24rem),
+      linear-gradient(180deg, rgba(255, 255, 255, 0.02), transparent 16rem);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)